.github/workflows/contrib.yml

```diff
@@ -25,14 +25,12 @@ jobs:
       run: |
         cd src/doc/contrib
         mdbook build
-        git worktree add gh-pages gh-pages
+        # Override previous ref to avoid keeping history.
+        git worktree add --orphan -B gh-pages gh-pages
```

Comment:
Could you verify that `gh-pages` doesn't exist in this environment? I did a rudimentary test and got this error:


```console
$ git worktree add --orphan -B gh-pages gh-pages
Preparing worktree (resetting branch 'gh-pages'; was at 8eb8acb)
fatal: a branch named 'gh-pages' already exist
```

Reply:
`-B` (rather than `-b`) explicitly overrides existing branches. I tested this on github-actions on my private fork and it correctly overrides existing branches. Did you get this result on a local test or on github-actions?

Reply:
> `-B` (rather than `-b`) explicitly overrides existing branches.

Yeah that's what I supposed. However, it didn't behave that way if my local repo has an existing `gh-pages`.

I'll merge this as is and see if it works. Thank you.

Reply:
Indeed. If you have a local branch, it fails. I guess the reason it worked in the workflow is because the `gh-pages` branch is not synced, so only exists as `origin/gh-pages`.

This might be a bug in git where `--orphan` and `-B` don't work together.